src/com/palmergames/bukkit/towny/TownyUniverse.java

```diff
@@ -44,6 +46,7 @@
     private final Hashtable<String, Town> towns = new Hashtable<>();
     private final Hashtable<String, Nation> nations = new Hashtable<>();
     private final Hashtable<String, TownyWorld> worlds = new Hashtable<>();
+    private final Hashtable<String, CustomDataField> registeredMetadata = new Hashtable<>();
```

Comment:
Is there any specific reason you used hashtable? I would go with a concurrent hashmap instead. Especially since most plugins will most likely access this from the main thread anyway.

Reply:
Good point